src/libraries/Native/Unix/System.Globalization.Native/pal_calendarData.c

```diff
@@ -128,15 +127,15 @@ int32_t GlobalizationNative_GetCalendars(
     char locale[ULOC_FULLNAME_CAPACITY];
     GetLocale(localeName, locale, ULOC_FULLNAME_CAPACITY, FALSE, &err);
     UEnumeration* pEnum = ucal_getKeywordValuesForLocale("calendar", locale, TRUE, &err);
-    int stringEnumeratorCount = uenum_count(pEnum, &err);
+    const int stringEnumeratorCount = uenum_count(pEnum, &err);
```

Comment:
Having local variables like this marked as `const` does not help with making the code easier to understand.

Reply:
Yeah, ok. But it does avoid holding a new value if someone by mistake overwrites it with another one, avoiding mistakes that one can do unknowingly. 😃

Reply:
That's true for variables with non-local context such as fields or methods arguments. I do not believe that it helps for local variables.